components/config-mgmt-service/backend/elastic/suggestions.go

```diff
@@ -21,6 +21,93 @@ func (es Backend) GetSuggestions(term string, text string, filters map[string][]
 	return es.getAggSuggestions(term, text, filters)
 }
 
+// {
+// 	"aggregations":{
+// 		 "myagg":{
+// 				"aggregations":{
+// 					 "distinct":{
+// 							"top_hits":{
+// 								 "_source":false,
+// 								 "size":1
+// 							}
+// 					 },
+// 					 "mymaxscore":{
+// 							"max":{
+// 								 "script":{
+// 										"source":"_score"
+// 								 }
+// 							}
+// 					 }
+// 				},
+// 				"terms":{
+// 					 "field":"platform",
+// 					 "order":[
+// 							{
+// 								 "mymaxscore":"desc"
+// 							}
+// 					 ],
+// 					 "size":100
+// 				}
+// 		 }
+// 	},
+// 	"query":{
+// 		 "bool":{
+// 				"filter":{
+// 					 "bool":{
+// 							"should":{
+// 								 "terms":{
+// 										"exists":[
+// 											 "true"
+// 										]
+// 								 }
+// 							}
+// 					 }
+// 				},
+// 				"must":[
+// 					 {
+// 							"type":{
+// 								 "value":"node-state"
+// 							}
+// 					 },
+// 					 {
+// 							"match":{
+// 								 "platform.engram":{
+// 										"operator":"or",
+// 										"query":"kk"
+// 								 }
+// 							}
+// 					 },
+// 					 {
+// 							"match":{
+// 								 "platform.engram":{
+// 										"operator":"and",
+// 										"query":"kk"
+// 								 }
+// 							}
+// 					 }
+// 				],
+// 				"should":[
+// 					 {
+// 							"term":{
+// 								 "platform.lower":{
+// 										"boost":200,
+// 										"value":"kk"
+// 								 }
+// 							}
+// 					 },
+// 					 {
+// 							"prefix":{
+// 								 "platform.lower":{
+// 										"boost":100,
+// 										"value":"kk"
+// 								 }
+// 							}
+// 					 }
+// 				]
+// 		 }
+// 	},
+// 	"size":0
+// }
```

Comment:
Adding this comment to make it easier to understand what the elasticsearch query is.